CharacterCreation/03_Feats.txt

```diff
@@ -294,320 +294,321 @@ upgrading it with skill points.
 
 Sleight of hand: 
 Prerequisites: DEX 6 or 15 Weapon Use[x] Skill
-Negate the pump of a shotgun, cocking of a revolver, or cycling of a bolt when
+	Negate the pump of a shotgun, cocking of a revolver, or cycling of a bolt when
 performing another action besides the aforementioned (besides firing). (Only applies
 to the weapons you have at least 15 Weapon Use[x] skill in using, unless you meet the
 DEX requirement).
 Reload DCs are 5% lower.
 
 Snap Shot: 
 Prerequisites: PER 6
-Time slows down slightly around you, or at least your perception is increased 
+	Time slows down slightly around you, or at least your perception is increased 
 allowing you to process everything around you more quickly for a split second. 
 When a target is within your weapon's R1 or R2, your miss chance is reduced
 by 1 (or 10% if relevant).
 
 Social Ninja: 
 Prerequisites: CHA 7
-Unless sneaking past / hiding from an alert enemy, use a Bluff/Deceive check in place 
+	Unless sneaking past / hiding from an alert enemy, use a Bluff/Deceive check in place 
 of a Stealth check.
 
 Strong Back:
 Prerequisites: STR 4
-Instead of a secondary weapon, you can carry an additional primary weapon.
+	Instead of a secondary weapon, you can carry an additional primary weapon.
 
 Pack Mule: 
 Prerequisites: STR 7, Strong Back
-You've done a lot of lifting and hauling in your time, allowing you to carry 
+	You've done a lot of lifting and hauling in your time, allowing you to carry 
 more than some. In addition to two primary weapons, you can carry another 
 secondary weapon.
 
-Tele-Cast
+Tele-Cast:
 Prerequisites: Paladin, Sorcerer, Mage.
-You may use a melee weapon to perform touch effect spells.
+	You may use a melee weapon to perform touch effect spells.
 
 Toughness: 
-Prerequisites:
-Roll a number of dice equal to your FORT / 5 (round up) + 1 and take the 
+Prerequisites: None
+	Roll a number of dice equal to your FORT / 5 (round up) + 1 and take the 
 result in extra hit points, or take the average of the result (1d10 = 5, 
 2d10 = 10, can be taken twice). If any of your die rolls is a 1 or 2, reroll
 the die.
 
 Upgrade: 
-Prerequisites:
-Roll a number of dice equal to your INT / 5 (round up) + 1 and take the result 
+Prerequisites: None
+	Roll a number of dice equal to your INT / 5 (round up) + 1 and take the result 
 in extra Nanites, or take the average of the result (1d10 = 5, 2d10 = 10, can 
 be taken twice). If any of your die rolls is a 1 or 2, reroll the die.
 
 Unconventional Warfare:
 Prerequisites: Any stat > 9, combat style must primarily use this stat.
-You are trained in the art of combative tossing, or some other obscure 
+	You are trained in the art of combative tossing, or some other obscure 
 fighting technique.
 
 Unlocked: Boomstick

```

Comment:
these 'Unlocked: Blah' feats are not going to display as section headings, similar to the ones that you updated well with the 'Acquired:blah' feats.

Reply:
This is not a blocking issue.